chunkset_tpl.h

```diff
@@ -152,21 +170,25 @@ Z_INTERNAL uint8_t* CHUNKMEMSET(uint8_t *out, unsigned dist, unsigned len) {
     }
 
     /* Last, deal with the case when LEN is not a multiple of SZ. */
-    if (rem)
+    if (rem) {
         memcpy(out, from, rem);
-    out += rem;
+        out += rem;
+    }
 
     return out;
 }
+Z_INTERNAL uint8_t* CSUFFIX(chunkmemset)(uint8_t *out, unsigned dist, unsigned len) {
+    return CSUFFIX(chunkmemset_static)(out, dist, len);
+}
 
-Z_INTERNAL uint8_t* CHUNKMEMSET_SAFE(uint8_t *out, unsigned dist, unsigned len, unsigned left) {
+Z_INTERNAL uint8_t* CSUFFIX(chunkmemset_safe)(uint8_t *out, unsigned dist, unsigned len, unsigned left) {
     if (left < (unsigned)(3 * sizeof(chunk_t))) {
```

Comment:
After this line value of `left` is ignored and as such code assumes `len` is never more than `left`. This is different from `chunkcopy_safe`, which always make sure there is enough room for `len` bytes.

Reply:
Not sure if I need to do anything here.